packages/styletron-engine-monolithic/src/client/client.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+
+/* eslint-env browser */
+
+import {StyleSheet} from "./sheet";
+
+import type {
+  StandardEngine,
+  KeyframesObject,
+  FontFaceObject,
+  StyleObject,
+} from "styletron-standard";
+
+import injectStylePrefixed from "../inject-style-prefixed.js";
+
+import {
+  keyframesBlockToRule,
+  declarationsToBlock,
+  keyframesToBlock,
+  fontFaceBlockToRule,
+  hashCssObject,
+} from "../css.js";
+
+declare var __DEV__: boolean;
+
+type hydrateT =
+  | HTMLCollection<HTMLStyleElement>
+  | Array<HTMLStyleElement>
+  | NodeList<HTMLStyleElement>;
+
+type optionsT = {
+  hydrate?: hydrateT,
+  container?: Element,
+  prefix?: string,
+};
+
+type cacheT = {
+  [key: string]: boolean,
+};
+
+class StyletronClient implements StandardEngine {
+  container: Element;
+  opts: optionsT;
+  fontFaceSheet: HTMLStyleElement;
+  keyframesSheet: HTMLStyleElement;
+
+  styleSheet: StyleSheet;
+  styleCache: cacheT;
+  keyframesCache: cacheT;
+  fontFaceCache: cacheT;
+
+  constructor(opts?: optionsT = {}) {
+    this.opts = opts || {};
```

Comment:
I still think it'd be nice to have a "strict/compat" mode that in development logs the same warnings as the atomic engine.